### PR TITLE
Add some remote mock methods.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
A bare minimum for mocking a cube for tests and starting point for discussion.

CC @benburkert

### CODE REVIEW

File: lib/fog/octocloud/requests/compute/remote_update_cube.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 module Fog
   module Compute
     class Octocloud
+      class Mock
+        def remote_update_cube(id, opts = {})
+          true
```
Comment (lstoll) [inline lib/fog/octocloud/requests/compute/remote_update_cube.rb:6]: This should merge the opts into data, no?

Comment (MikeMcQuaid) [inline lib/fog/octocloud/requests/compute/remote_update_cube.rb:6]: Yup; wasn't sure how `id` related to `data`.

Comment (lstoll) [inline lib/fog/octocloud/requests/compute/remote_update_cube.rb:6]: It's something we need to have for looking up Items. Basically the 'name' should be the ID, I think that's shitty inconsistencies on my part.

**Comment by lstoll** [comment]
I'll merge this now, because it adds enough to get you by.

I'm going to do a refactor on the requests section, I hate the one file per request model, and some of the inconsistent naming. I'll add additional functionality then.

**Comment by MikeMcQuaid** [comment]
Nice one, cheers.

**Pull request merged**